sequential/seq2pat.py

```diff
@@ -382,11 +386,16 @@ def get_patterns(self, min_frequency: Num) -> List[list]:
 
```

Comment:
don't we need a check_true (self._num_rows >= 1) independent of if (float) else (int) logic?

Reply:
> don't we need a check_true (self._num_rows >= 1) independent of if (float) else (int) logic?

I think this might be covered already. We checked the input needs to be a list of list, and input has no empty list. But this can be added to make it explicit.

Reply:
Added a check when sequences/attributes are initialized with an empty list, and also have `num_rows` checked.